packages/qdrant-loader/src/qdrant_loader/connectors/confluence/connector.py

```
@@ -792,162 +804,256 @@ def _clean_html(self, html: str) -> str:
-                try:
-                    page_count += 1
-                    logger.debug(
-                        f"Fetching page {page_count} of Confluence content (cursor={cursor})"
-                    )
-                    response = await self._get_space_content_cloud(cursor)
-                    results = response.get("results", [])
+        while True:
+            try:
+                page_count += 1
+                logger.debug(
+                    f"Fetching page {page_count} of Confluence content (cursor={cursor})"
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)